index.js

```diff
@@ -124,7 +126,7 @@ const getTreeFromYarnLock = ({ yarnLock: yarnLockString, packageJSON }) => {
   return tree
 }
 
-const getTreeFromNodeModules = async dir => {
+const getTreeFromNodeModules = async (dir, { dev = true } = {}) => {
```

Comment:
nitpick: avoid optional options if possible.

Reply:
i think its fine for `dev = true` to be optional; but ideally all callers to `getTreeFromNodeModules` pass in `{ dev: bool }` as a required argument.

Too many optional parameters leads to accidentally not passing opts.dev down to the next function call.